pkg/apis/common/v1/types.go

```diff
@@ -128,6 +128,11 @@ const (
 	// reached phase failed with no restarting.
 	// The training has failed its execution.
 	JobFailed JobConditionType = "Failed"
+
+	// JobReconcileFinished means this job reached JobSucceeded or JobFailed
+	// and its sub-resources have been cleaned according to CleanPodPolicy,
+	// No more reconciliation is needed for this job.
+	JobReconcileFinished JobConditionType = "ReconcileFinished"
```

Comment:
Name ReconcileFinished  here is confusing to me. BTW, does 12h or 24h reconcile period works for you?

Reply:
Do you have a suggestion for the name of the status? 
I never try using 12h or 24h reconcile period because in my environment I implemented another fix for https://github.com/kubeflow/tf-operator/issues/956, which requires reconcile period to be short.